src/core/types/api/dashboard.ts

```diff
@@ -26,13 +26,20 @@ export type DashboardStatsType = {
 };
 
 export type DashboardEpisodeDetailsType = {
-  ID: number,
+  IDs: {
+    ID: number,
+    Series: number | null,
```

Comment:
we use Series?: number; instead of | null;

Reply:
But it will be a number or null, not a number or undefined. Doing a(n accidental) strict equal to undefined would result in false if the value is null.
Also, the (anidb) series id will always be set and never be null. Sorry,  my (1am) mistake, will fix it now.

Reply:
I thought it was null, but if it is undefined then leave it as is.